src/renderer/services/binance/service.ts

```diff
@@ -337,9 +337,11 @@ const fees$: Observable<FeesRD> = client$.pipe(
 
 /**
  * Filtered fees to return `TransferFees` only
- * If a client is not available, it returns `None`
  */
-const transferFees$: Observable<TransferFeesRD> = fees$.pipe(map(RD.map(getTransferFees)), shareReplay(1))
+const transferFees$: Observable<O.Option<TransferFees>> = fees$.pipe(
+  switchMap((fees) => Rx.of(FP.pipe(fees, RD.toOption, O.map(getTransferFees), O.flatten))),
```

Comment:
have 2 questions here:
1. why do we need to have an Option instead of RD?
2. why do we need to `swicthMap` here? we just map a data here

Reply:
1) My idea was to simplify things for views, since views do not need to know about any loading state of fees (currently). However, it can be changed in the future, so I will use RD here (_as you suggested - thanks!_).
2) You are right!